### PR TITLE
Feature/ffp 479 chat v3

### PR DESCRIPTION
**Pull request by recursive-franciscop**
Added markdown and syntax highlighting to the chat:

image

### CODE REVIEW

File: web/src/types/Stream.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+type Stream =
+  | DeltaStream
+  | FullStream
+  | EmptyStream
+  | FinishStream
+  | StartStream
+  | EndStream
+  | InfoStream;
+export default Stream;
```
Comment (recursive-franciscop) [inline web/src/types/Stream.ts:49]: @miguelnmiranda and @antoniofpnmartins I'm going to need your review here, could you please point to me where I could find more info of the types of streaming messages?

Comment (antoniofpnmartins) [reply web/src/types/Stream.ts:49]: https://github.com/recursiveai/findflow_fenix/blob/main/api/src/recursiveai/findflow/assistant/api/schemas/chat.py, lines ~55 to 95

Comment (recursive-franciscop) [reply web/src/types/Stream.ts:49]: Oh that seems perfect! I'm only missing the "references" and "file_selection_needed" types from the data+controls, and assistant_selected, audit_id and pre_selected_files from keys in the "info" controls

**Review by miguelnmiranda — APPROVED** [review]
LGTM

**Comment by recursive-franciscop** [comment]
Good to go now! Let's wait for the CI to finish and then can merge

**Pull request merged**